Source/WebCore/platform/ios/VideoPresentationInterfaceIOS.mm

```diff
@@ -249,6 +249,21 @@ @interface UIViewController ()
     m_playbackSessionInterface->setPlayerIdentifier(WTFMove(identifier));
 }
 
+void VideoPresentationInterfaceIOS::audioSessionCategoryChanged(WebCore::AudioSessionCategory, WebCore::AudioSessionMode, WebCore::RouteSharingPolicy)
+{
+    auto model = videoPresentationModel();
```

Comment:
`auto` -> `RefPtr` as I understand we don't want to hide smart pointers behind auto in new code